modules/dailymotionBidAdapter.md

```diff
@@ -102,15 +116,18 @@ const adUnits = [
         context: 'instream',
         description: 'this is a video description',
         duration: 556,
+        iabcat1: ['IAB-2'],
         iabcat2: ['6', '17'],
         id: '54321',
         lang: 'FR',
-        playerSize: [ [1280, 720] ],
+        livestream: 0,
         private: false,
-        startDelay: 0,
+        startdelay: 0,
         tags: 'tag_1,tag_2,tag_3',
         title: 'test video',
         topics: 'topic_1, topic_2',
```

Comment:
Please exclude fields that are not defined in `imp[].video` from `mediaTypes.video`. 
The direction we are moving is to align `imp[].video` with `mediaTypes.video` to simplify onboarding for publishers. Most publishers use multiple SSPs; by standardizing the configuration across the board, we can reduce the complexity of getting setup with video in Prebid. 

Everything outside of the spec can be set in `params.video`

Reply:
Dropped all support of non standard fields in `mediaTypes.video`. Updated documentation accordingly